src/Common/FileCacheType.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <Core/Types.h>
+
+namespace DB
+{
+
+struct FileCacheKey
+{
+    UInt128 key;
+    String toString() const;
```

Comment:
Let's move Implementation of this method (`toString`) into this file, it is in some of the other cache files, which does not seem convenient now -- not obvious where to look for.